blockchain/state_manager/src/chain_rand.rs

```diff
@@ -12,18 +13,27 @@ use encoding::blake2b_256;
 use forest_blocks::{Tipset, TipsetKeys};
 use forest_crypto::DomainSeparationTag;
 use interpreter::Rand;
-use std::error::Error;
+// use std::error::Error;
```

Comment:
I'd like to keep the uncommented  code in this file as an easily accessible record of how `Rand` worked in the past. We'll eventually have to restore the code. Guess this would be noted in the comments.

Reply:
Ah, I guess we're still keeping all the `*_v1` / `*_v2` functions. I'll remove the disabled code.